search.html

```diff
@@ -0,0 +1,188 @@
+            with a mosquito.</q
+          >
+          <p>~ Dalai Lama</p>
+        </div>
+        <div class="post__icons">
+          <span class="post__icon--thumb mr-1rem">2,245</span>
+          <span class="post__icon--bubble mr-1rem">45</span>
+          <span class="post__icon--share mr-1rem">124</span>
+        </div>
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          data-prefix="far"
+          data-icon="bookmark"
+          class="post__bookmark svg-inline--fa fa-bookmark"
+          role="img"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 384 512"
+        >
+          <path
+            fill="currentColor"
+            d="M336 0h-288C21.49 0 0 21.49 0 48v431.9c0 24.7 26.79 40.08 48.12 27.64L192 423.6l143.9 83.93C357.2 519.1 384 504.6 384 479.9V48C384 21.49 362.5 0 336 0zM336 452L192 368l-144 84V54C48 50.63 50.63 48 53.1 48h276C333.4 48 336 50.63 336 54V452z"
+          ></path>
+        </svg>
+      </section>
+      <section class="post">
+        <div class="avatar">
+          <img
+            class="avatar__image"
+            src="https://source.unsplash.com/100x100/?faces"
+            alt=""
+          />
+          <h2 class="avatar__heading">Tony Monta</h2>
+          <span class="avatar__timespan">1h ago</span>
+        </div>
+        <span class="post__burger-menu"></span>
+        <div class="post__text">
+          <p>Writing code is not so bad!</p>
+        </div>
+        <div class="post__icons">
+          <span class="post__icon--thumb mr-1rem">14</span>
+          <span class="post__icon--bubble mr-1rem"></span>
+          <span class="post__icon--share mr-1rem"></span>
+        </div>
+        <svg
+          aria-hidden="true"
+          focusable="false"
+          data-prefix="far"
+          data-icon="bookmark"
+          class="post__bookmark svg-inline--fa fa-bookmark"
+          role="img"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 384 512"
+        >
+          <path
+            fill="currentColor"
+            d="M336 0h-288C21.49 0 0 21.49 0 48v431.9c0 24.7 26.79 40.08 48.12 27.64L192 423.6l143.9 83.93C357.2 519.1 384 504.6 384 479.9V48C384 21.49 362.5 0 336 0zM336 452L192 368l-144 84V54C48 50.63 50.63 48 53.1 48h276C333.4 48 336 50.63 336 54V452z"
+          ></path>
+        </svg>
+      </section>
+    </main>
+
+    <footer class="footer navigation">
+      <i class="fa fa-home fa-lg"><span class="sr-only">Home</span></i>
+      <i class="navigation__icon--grey fa fa-search fa-lg"
+        ><span class="sr-only">Search</span></i
+      >
+      <i class="navigation__icon--plus fa fa-plus fa-lg"
+        ><span class="sr-only">Plus</span></i
+      >
+      <i class="navigation__icon--grey fa fa-bell fa-lg"
+        ><span class="sr-only">Bell</span></i
+      >
+      <i class="navigation__icon--grey fa fa-user-circle fa-lg"
+        ><span class="sr-only">Profile</span></i
+      >
+    </footer>
```

Comment:
This should be a `nav` as it is for navigation. It should have `buttons` (or anchors for now) for interaction.